### PR TITLE
Adding Ads with Navegg

### PR DESCRIPTION
**Pull request by chaveznvg**

### CODE REVIEW

File: ads/navegg.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/navegg.js]: 2017

Comment (zhouyx) [inline ads/navegg.js]: 2017

File: ads/navegg.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/navegg.md]: 2017

File: examples/navegg-add.amp.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<!doctype html>
```
Comment (zhouyx) [inline examples/navegg-add.amp.html]: you don't need to add an example page, please only add the example to `examples/ads.amp.html`

File: ads/navegg.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Navegg
+
+If you want to serve ads with Navegg data you need to use this adapter.
+
+The Navegg adapter only supports Doubleclick for now.
+
+## Example
+
+```html
+<amp-ad width=320 height=50
```
Comment (zhouyx) [inline ads/navegg.md]: please re format the example here. 2 space indentation, and `>` at the end of line.

File: ads/navegg.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+import {loadScript} from '../3p/3p';
+import {doubleclick} from '../ads/google/doubleclick';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function navegg(global, data) {
+  const acc = data.acc;
```
Comment (zhouyx) [inline ads/navegg.js]: question: is `data.acc` required? what if it is undefined? If it is required, please update in the doc, also use `validateData` to valid it.

File: ads/navegg.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  let seg, nvg = function() {};;
```
Comment (zhouyx) [inline ads/navegg.js]: nit: double ;

File: ads/navegg.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!---
```
Comment (zhouyx) [inline ads/navegg.md]: please link the doc from amp-ad.md.

**Comment by chaveznvg** [comment]
I signed it!

**Comment by chaveznvg** [comment]
I signed it!

**Pull request closed without merging**